doc/going-further/compile.md

```diff
@@ -0,0 +1,51 @@
+# Compiling your application into a standalone binary
+
+[Putting your Castor application into a PHAR archive](repack.md) can be a good way to easily share and use it in various environments.
+
+However, you need to ensure that PHP is installed and configured correctly in all the environments where you want to use your Castor app. 
+This can be a hassle, especially if you don't have control over the environments.
+
+To make things simpler, Castor's `compile` command can help by creating a customizable PHP binary with a PHAR archive, making one executable file that can be used in any setting. 
+
+Just pass your repacked Castor app PHAR as an argument of this command.
+
+## Pre-requisites
+
+Follow the [`repack` documentation](repack.md) to output a PHAR archive of your Castor app.
+
+## Running the Compile Command
+
+To compile your Castor application, navigate to your project directory and run:
+
+```bash
+vendor/bin/castor compile my-custom-castor-app.phar
+```
+
+> [!WARNING]
+> Compiling is not supported yet on Windows.
+
+### Options
+
+Make sure to take a look at the command description to see all the available options:
+```bash
+vendor/bin/castor compile -h
```

Comment:
```suggestion
vendor/bin/castor compile --help
```

In documentation, and commited scripts, I always prefer long options.
`-h` is quite easy, but it's a matter of habits :)